src/pages/NewBeer.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+
+import NavBar from '../components/NavBar'
+import { useState } from 'react'
+import axios from 'axios'
+import { useNavigate } from "react-router-dom"
+function NewBeer() {
+    const [name, setName] = useState('')
+    const [tagline, setTagline] = useState('')
+    const [description, setDescription ] = useState('')
+    const [firstBrewed, setFirstBrewed ]  = useState('')
+    const [brewersTips, setBrewersTips] = useState('')
+    const [attenuationLevel, setAttenuationLevel] = useState('')
+    const [contributedBy, setContributedBy ] = useState('')
+    const navigate = useNavigate(); 
+    const handleSubmit = (e) => {
+        e.preventDefault()
+        const body = {
+            name: name,
+             tagline: tagline,
+              description: description,
+              first_brewed:firstBrewed,
+               brewers_tips:brewersTips,
+                attenuation_level:attenuationLevel,
+                contributed_by:contributedBy}
+        axios
+        .post('https://ih-beers-api2.herokuapp.com/beers/new', body)
+        .then((res)=>{
+            setName('')
+            setTagline('')
+            setDescription('')
+            setFirstBrewed('')
+            setBrewersTips('')
+            setAttenuationLevel('')
+            setContributedBy('')
+            navigate('/');    
+        })
+        .catch((error) => {
+            console.error('Error creating beer:', error);
+        });
+        
+    }
+
+  return (
+    <div>
+        <NavBar></NavBar>
+        
+        <h2>New Beer</h2>
+        <form onSubmit={handleSubmit}>
+            <label >Name</label>
+            <input type="text" name='name' onChange={(e)=> setName(e.target.value) } value={name}/>
```

Comment:
I would wrap the label on the input to give a little bit more readability for screen readers and also add a `required` attribute on the input for the name (is the only one required for that API) so that would help a little bit the user understand which one is needed